sdk/cosmos/azure-cosmos/test/test_full_text_policy.py

```diff
@@ -0,0 +1,264 @@
+        cls.test_db = cls.client.create_database(str(uuid.uuid4()))
+
+    def test_create_full_text_container(self):
+        # Create a container with a valid full text policy and full text indexing policy
+        full_text_policy = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        indexing_policy = {
+            "fullTextIndexes": [
+                {"path": "/abstract"}
+            ]
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+        self.test_db.delete_container(created_container.id)
+
+        # Create a container with a full text policy containing only default language
+        full_text_policy_no_paths = {
+            "defaultLanguage": "en-US"
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy_no_paths,
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy_no_paths
+        self.test_db.delete_container(created_container.id)
+
+    def test_replace_full_text_container(self):
+        # Replace a container with a valid full text policy and full text indexing policy
+        full_text_policy = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        indexing_policy = {
+            "fullTextIndexes": [
+                {"path": "/abstract"}
+            ]
+        }
+        created_container = self.test_db.create_container(
+            id='full_text_container' + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/id"),
+            full_text_policy=full_text_policy,
+            indexing_policy=indexing_policy
+        )
+        properties = created_container.read()
+        assert properties["fullTextPolicy"] == full_text_policy
+        assert properties["indexingPolicy"]['fullTextIndexes'] == indexing_policy['fullTextIndexes']
+
+        # Replace the container with new policies
+        full_text_policy['fullTextPaths'][0]['path'] = "/new_path"
+        indexing_policy['fullTextIndexes'][0]['path'] = "/new_path"
+        replaced_container = self.test_db.create_container(
```

Comment:
In the doc of `create_container`, it says "If a container with the given ID already exists, a CosmosResourceExistsError is raised." 

Is it not true anymore? 

I don't see any change enabling `replace` behavior

Reply:
yep that was my bad, used the wrong method